x11-doc/xorg_sgml_doctools/xorg_sgml_doctools-1.11.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A common set of SGML entities and XML/CSS style sheets"
+DESCRIPTION="This package provides a common set of SGML entities and XML/CSS \
+style sheets used in building/formatting the documentation provided in other\
+X.Org packages.   It's typically only needed by people building from \
+source who want to produce formatted documentation from their builds, \
+or those who have installed the HTML version of the documentation, \
+which refers to the included common xorg.css stylesheet."
+HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
+COPYRIGHT="2010, Oracle and/or its affiliates
+	2005 Red Hat, Inc."
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/freedesktop/xorg-sgml-doctools/archive/xorg-sgml-doctools-$portVersion.tar.gz"
+CHECKSUM_SHA256="2dfe05eb06435f142da3adfae76503ed96cd4f3c2e2541a6ca89037ba07460bb"
+SOURCE_DIR="xorg-sgml-doctools-xorg-sgml-doctools-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	xorg_sgml_doctools = $portVersion
+	devel:xorg_sgml_doctools = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	devel:util_macros
+	cmd:aclocal
+	cmd:autoconf
+	cmd:gcc
+	cmd:ld
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	autoreconf -vfi
+	runConfigure ./configure
+}
+
+INSTALL()
+{
+	make install
+	fixPkgconfig
```

Comment:
If the `.pc` files are in `dataDir` (it have to if we want to classify this as an `any` package), then we can remove `fixPkgconfig` from the recipe.